nipype/algorithms/tests/test_misc.py

```diff
@@ -8,7 +8,6 @@
 
 from nipype.algorithms import misc
 from nipype.utils.filemanip import fname_presuffix
-from nipype.testing.fixtures import create_analyze_pair_file_in_directory
```

Comment:
You've removed a lot of fixtures from the module namespaces. This could be generally rectified by either moving or importing fixtures from `nipype.testing.fixtures` into `nipype.conftest`.

Reply:
I've put them back for now.